src/core.js

```diff
@@ -348,6 +348,16 @@ extend( QUnit, {
 				return type.toLowerCase();
 		}
 		if ( typeof obj === "object" ) {
+			if ( obj instanceof Int8Array ||
```

Comment:
Looking at the failing tests in IE, you need a feature detect before referencing any of these. Also applies to the unit tests.